service/provider/src/main/java/gov/va/vro/service/provider/services/DiagnosisLookup.java

```diff
@@ -7,7 +7,7 @@ public final class DiagnosisLookup {
   private DiagnosisLookup() {}
 
   private static final Map<String, String> diagnosisMap =
-      Map.of("7101", "Hypertension", "6602", "Asthma");
+      Map.of("7101", "Hypertension", "6602", "Asthma", "1233", "1233");
```

Comment:
can you try changing the test that says 1233 instead, making that 6602 and 7101?

Reply:
One of them tests if the claim is out of scope and needs to be offramped, so I think Dimitri did this on purpose.  I did fix one of the tests though.

Reply:
Is 1233 an actual value or just for testing? It it is for testing, it should not appear in the real Lookup